electrum_dash/coins.py

```diff
@@ -16,6 +16,9 @@ class Firo(Coin):
 
     @classmethod
     def static_header_offset(cls, height):
+        if (cls.__name__== 'FiroRegtest'):
+            return cls.PRE_MTP_HEADER_SIZE * height
```

Comment:
Don't we have the same effect by setting `PRE_MTP_BLOCKS` to `INT_MAX`? These ifs aren't looking great.